DOC-2194: added fix documentation for TINY-10263 to the 6.7.2 Release notes (#2952)
tinymce/tinymce-docs: modules/ROOT/pages/6.7.2-release-notes.adoc

=== The function `getModifierState` did not work on events passed through the editor as expected.
// TINY-10263

> The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when the editor called the 'getModifierState' function on a **keydown** event action.

Reviewer: It's the integrator or developer who calls the function and that causes an error. I don't think we even use that function that is why we missed it not being executed with the right context.

```suggestion
The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when an integrator called the 'getModifierState' function on a **keydown** event action.
```

Reviewer 2: The method `getModeierState` is called on an event object,  not an event action.

> This error was introduced, when the function was re-used in a previous update to {productname}, but, the updated logic for the origional function was not changed to consider the newly introduced context.

Reviewer: ```suggestion
This error was introduced, when the function was re-used in a previous update to {productname}, but, the updated logic for the original function was not changed to consider the newly introduced context.
```

> This error was introduced, when the function was re-used in a previous update to {productname}, but, the updated logic for the original function was not changed to consider the newly introduced context.

Author: @spocke suggest we remove this
```suggestion
```

> As a consequence, any attempts to invoke this function resulted in the TypeError being displayed in the users developer console.

Author: ```suggestion
As a consequence, any attempts to invoke this function resulted in the TypeError being displayed in the console.
```

> {productname} 6.7.2 addresses this issue, by updating the 'getModifierState' function to now consider the updated context introduced in a previous release.

Author: @HAFRMO I agree, but im not sure if providing less info will be sufficient enough for a docs entry, as I dont believe telling the customer that we "copied over a function without concern of context" is a good idea either as mentioned in TINY-10263 docs outline? as this might give them the wrong impression?

> {productname} 6.7.2 addresses this issue, by updating the 'getModifierState' function to now consider the updated context introduced in a previous release.

Reviewer: I think we just need to explain that an exception was thrown if the integrator called `getModifierState` on the event object that the editor provide though the `keydown` event handler call. How to explain that in simpler terms I'm not sure.

> {productname} 6.7.2 addresses this issue, by updating the 'getModifierState' function to now consider the updated context introduced in a previous release.

Reviewer 3: I think Spocke's right on this one.

> {productname} 6.7.2 addresses this issue, by updating the 'getModifierState' function to now consider the updated context introduced in a previous release.

Author: ```suggestion
{productname} 6.7.2 addresses this, now, when a integrator uses the `getModifierState` on the event object a keydown event no longer triggers an error.
```

> As a result, the TypeError is no longer displayed when the editor calls the 'getModifierState' function on an event during a **keydown** action.

Author: ```suggestion
```
